.gitignore

```diff
@@ -2,4 +2,5 @@
 .idea/
 node_modules/
 Pulumi.*.yaml
-!Pulumi.sample.yaml
\ No newline at end of file
+!Pulumi.sample.yaml
+dist/
```

Comment:
this is better than the copy, but I was actually thinking since we have a `.gitignore`, `node/.gitignore`, and `go/.gitignore`, just merging them all into `unchained/.gitignore`. i thought it might be nice to keep the rules separate, but I don't think there is actually any gain and just makes management more complicated